.gitlab-ci.yml

```diff
@@ -59,4 +59,5 @@ publish:
         - echo '//registry.npmjs.org/:_authToken=${NPM_TOKEN}'>.npmrc
         - npm publish
     only:
+        - tags
         - /\d+\.\d+\.\d+/
```

Comment:
this will still match so you can probably remove this line now

Reply:
docs were a little confusing but I thought this would only run if both conditions were met...
I'll remove then